### PR TITLE
[BACKLOG-38558] SFTP PUT Core To Plugin Movement

### PR DESCRIPTION
**Pull request by Mandakini111**

### CODE REVIEW

File: plugins/sftpput/ui/src/main/java/org/pentaho/di/ui/trans/steps/sftpput/SFTPPutDialog.java [hunk 1]
```diff
@@ -206,7 +210,7 @@ public class SFTPPutDialog extends BaseStepDialog implements StepDialogInterface
 
   private SFTPClient sftpclient = null;
 
-  public SFTPPutDialog( Shell parent, Object in, TransMeta tr, String sname ) {
+  public SFTPPutDialog(Shell parent, Object in, TransMeta tr, String sname ) {
```
Comment (addagudi) [inline plugins/sftpput/ui/src/main/java/org/pentaho/di/ui/trans/steps/sftpput/SFTPPutDialog.java:213]: Please fix the spacing according to pentaho-coding-standards. 
Refer https://github.com/pentaho/pentaho-coding-standards

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 58m 54s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/plugins,engine,plugins,plugins/sftpput,plugins/sftpput/assemblies,plugins/sftpput/assemblies/plugin,plugins/sftpput/impl,plugins/sftpput/ui,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **5324**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9009/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/sftpput/ui/src/main/java/org/pentaho/di/ui/trans/steps/sftpput/SFTPPutDialog.java (hunk 1 above)
Comment (Mandakini111) [reply plugins/sftpput/ui/src/main/java/org/pentaho/di/ui/trans/steps/sftpput/SFTPPutDialog.java]: Fixed the issue

**Review by addagudi — APPROVED** [review]

**Comment by AAravind27** [comment]
You have merge conflict in `assemblies/plugins/pom.xml`

**Pull request closed without merging**